packages/flutter/test/cupertino/switch_test.dart

```diff
@@ -1867,6 +1867,50 @@ void main() {
       imageProvider2.complete();
       expect(tester.takeException(), isNull);
     });
+
+    testWidgets('Switch uses inactive track color when set', (WidgetTester tester) async {
+        const Color inactiveTrackColor = Color(0xFF00FF00);
```

Comment:
Nit: The contents of both tests should probably be outdented.

Reply:
Ah good catch. I think it's ok to land this, we can tidy it up after. The tag on the deprecation is time sensitive.